RHEL/7/input/xccdf/system/software/sudo.xml

```diff
@@ -0,0 +1,65 @@
+<Group id="sudo">
+<title>Sudo</title>
+<description>
+<tt>Sudo</tt>, which stands for "su 'do'", provides the ability to delegate authority
+to certain users, groups of users, or system administrators. When configured for system 
+users and/or groups, <tt>Sudo</tt> can allow a user or group to execute privileged commands
+that normally only <tt>root</tt> is allowed to execute.
+<br/><br/>
+For more information on <tt>Sudo</tt> and addition <tt>Sudo</tt> configuration options, see
+<b>https://www.sudo.ws</b>
+</description>
+
+<Rule id="sudo_remove_nopasswd" severity="medium">
+<title>Remove nopasswd Usage From Sudo</title>
+<description>
+The sudo <tt>NOPASSWD</tt> tag, when specified, allows a user to execute commands using
+sudo without having to authenticate. This should be disabled by making sure that the
+<tt>NOPASSWD</tt> tag does not exist in <tt>/etc/sudoers</tt> configuration file or 
+any sudo configuration snippets in <tt>/etc/sudoers.d</tt>.
+</description>
+<ocil clause="nopasswd is enabled in sudo">
+To determine if <tt>NOPASSWD</tt> has been configured for sudo, run the following command:
+<pre>$ sudo grep NOPASSWD /etc/sudoers /etc/sudoers.d/</pre>
```

Comment:
Attempt to grep a directory without ```-r``` option will issue an error:
```
$ grep 'test' /etc/sudoers.d
grep: /etc/sudoers.d: Is a directory
```
If you want to grep both locations above, ```-r``` option should be specified (since modules present there should be searched).